plugins/inputs/mysql/v2/convert.go

```diff
@@ -53,6 +54,29 @@ func ParseString(value sql.RawBytes) (interface{}, error) {
 	return string(value), nil
 }
 
+// ParseWsrepLatency parses the given sql.RawBytes value into a map
+// containing 5 distinct float64 values. These represent min/avg/max/stdev/sample_size.
+// It returns an error if the value is unrecognized.
+func ParseWsrepLatency(value sql.RawBytes) (interface{}, error) {
+	parts := strings.Split(string(value), "/")
+	if len(parts) != 5 {
+		return nil, fmt.Errorf("unsupported amount of values in wsrep_evs_repl_latency, got %d, should be 5", len(parts))
+	}
+
+	result := make(map[string]interface{})
+	keys := []string{"min", "avg", "max", "stdev", "sample_size"}
+
```

Comment:
How about pre-allocating the result-map?

```suggestion
	keys := []string{"min", "avg", "max", "stdev", "sample_size"}
	result := make(map[string]interface{}, len(keys))
```